common/event/base_event.py

```
@@ -0,0 +1,169 @@
+from common.utils import get_utc_now
+
+logger = get_logger(__name__)
+
+
+class BaseEvent:
+    table_name: str = None
+    old: dict = {}
+    new: dict = {}
+    id: str = None
+    action: str = None
+    diff: dict = {},
```

Pylint (3.3.7)

[refactor] Disallow trailing comma tuple

(R1707)